### PR TITLE
Added Ambulance module

### PR DESCRIPTION
**Pull request by luluCommitter**

### CODE REVIEW

File: HospitalManagement/src/main/java/com/realhospitalmanagementsystem/sevice/seviceImpl/AmbulanceServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
+package com.realhospitalmanagementsystem.sevice.seviceImpl;
+
```
Comment (AMAR-sabar) [inline HospitalManagement/src/main/java/com/realhospitalmanagementsystem/sevice/seviceImpl/AmbulanceServiceImpl.java:2]: service package name mismatch "sevice"

File: HospitalManagement/src/main/java/com/realhospitalmanagementsystem/entity/Ambulance.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+@AllArgsConstructor
+@Table(name="Ambulance_Details")
+public class Ambulance {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+@Column(name="RegistrationNumber" , nullable = false)
+    private String registrationNumber;
+    @Column(name="DriverName" , nullable = false)
+    private String driverName;
```
Comment (AMAR-sabar) [inline HospitalManagement/src/main/java/com/realhospitalmanagementsystem/entity/Ambulance.java:20]: add one more field like emergency number.

**Review by AMAR-sabar — APPROVED** [review]

**Comment by AKASHBARDE** [comment]
Got it, #15 has been successfully merged into Main-master. Thank you for
the update.

**Comment by AKASHBARDE** [comment]
thnkyou

**Pull request merged**